packages/core/components/ListPicker/index.tsx

```diff
@@ -155,6 +161,17 @@ export default function ListPicker(props: ListPickerProps) {
                             Clear all
                         </ActionButton>
                     </Tooltip>
+                    {shouldShowNullGroups !== undefined && ( // avoid colliding with falsy value
```

Comment:
Why would `shouldShowNullGroups` be undefined?

Reply:
We use the ListPicker component in multiple places, and for some of them we don't want the checkbox to show up (e.g., Sort). I'm double-using the `shouldShowNullGroups` property here by making it optional -- When we want the checkbox to show up, it's set to either true or false (which also coincides with how the checkbox should be set). When we don't want it to show up, I leave off the prop and it stays undefined